### PR TITLE
88 remove old code

### PR DESCRIPTION
**Pull request by maho3**
Removes various legacy code that was either used in early versions of ltu-ili or for private projects. Addresses the following tasks from Issue #88 :

- Remove any temporary scripts from the main branch (e.g. the ones used for private projects).
- Remove the Cosmology object and related tests.

Note, all this code is being removed from the main branch, but is preserved in branch [backup_2023-20-11](https://github.com/maho3/ltu-ili/tree/backup_2023-20-11).

### CODE REVIEW

File: tests/run.sh
Comment (DeaglanBartlett) [inline tests/run.sh:1]: Line 19
`coverage combine .coverage_sbi .coverage_pydelfi .coverage_utils`
needs changing to
`coverage combine .coverage_sbi .coverage_pydelfi`
as `.covergae_utils` is no longer created

**Review by DeaglanBartlett — CHANGES_REQUESTED** [review]
Looks good to me, except one small required change in tests/run.sh as we no longer create the file .coverage_utils so cannot merge this file with others when merging the coverage reports.

File: tests/run.sh
Comment (maho3) [reply tests/run.sh:1]: Thanks missed this! Removed.

**Comment by maho3** [comment]
Note, this PR now also addresses "Make Nicolas' test notebooks consistent with the new version of ltu-ili, or otherwise remove them" from Issue #88 .

**Review by DeaglanBartlett — APPROVED** [review]

**Pull request merged**
